framework/doc/content/source/meshgenerators/AllSimplexGenerator.md

```diff
@@ -0,0 +1,30 @@
+# AllSimplexGenerator
+
+!syntax description /Mesh/AllSimplexGenerator
```

Comment:
I have a slight preference for Converter over Generator considering the [ElementsToTetrahedronsConverter](https://mooseframework.inl.gov/source/meshgenerators/ElementsToTetrahedronsConverter.html)

prefer AllSimplex... or ElementsToSimplicesConverter ? or SimplexConversionGenerator?
some ideas. We have not been naming MGs after the libMesh all_do_something routines 

which btw we could deprecate?